third-party/cmake/BuildLibtermkey.cmake

```diff
@@ -1,5 +1,3 @@
-find_package(PkgConfig REQUIRED)
-
```

Comment:
In `third-party/CMakeLists.txt` we pull from a different libtermkey repo:

```
if(WIN32)
set(LIBTERMKEY_URL https://github.com/equalsraf/libtermkey/archive/tb-windows.zip)
set(LIBTERMKEY_SHA256 c81e33e38662b151a49847ff4feef4f8c4b2a66f3e159a28b575cbc9bcd8ffea)
else()
...
```

to get this patch https://github.com/equalsraf/libtermkey/commit/b9317d6473b7a598359ebdfae5fd228bc159d0de

We could instead include that patch using `PATCH_COMMAND` as you've done with the others.